src/libraries/System.Reflection.MetadataLoadContext/src/System/Reflection/TypeLoading/Types/RoType.cs

```diff
@@ -191,10 +191,9 @@ private RoType[] ComputeInterfaceClosure()
                     }
                 }
             }
-
-            // todo: use IEnumerable<T> extension: return ifcs.ToArray()
-            List<RoType> list = new List<RoType>(ifcs);
-            return list.ToArray();
+            RoType[] arr = new RoType[ifcs.Count];
```

Comment:
```suggestion

            RoType[] arr = new RoType[ifcs.Count];
```

Reply:
@stephentoub, Does this improve performance or memory usage in some way?

Reply:
`List<T>.ToArray()` special-cases a length of 0 to avoid allocating a new array and just returning a zero-length singleton.  The original change was losing that.  This adds it back.